tests/roles/dataplane_adoption/tasks/main.yaml

```diff
@@ -356,8 +368,10 @@
         - run-os
         - install-certs
         - ceph-client
+        {%+ if compute_adoption|bool +%}
```

Comment:
this should be somehow aligned with docs, maybe based on ENV vars, or if-else sections?

Reply:
@bogdando could you please clarify what you mean? The docs have a separate section for Networker nodes, and this section doesn't list the libvirt and nova-compute* services in the example snippet. Anything else that I am missing?

Reply:
There is a rule that code in this repo should look the same way in docs. @jistr WDYT for this case, how should we proceed?